app/views/courses/vacancies/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= content_for :page_title, "Vacancies" %>
 
-<%= manage_ui_link_to_back("/organisation/#{params[:provider_code]}/courses/#{@course.course_code}") %>
+<%= govuk_back_link_to(manage_ui_course_page_url(provider_code: params[:provider_code], course_code: @course.course_code)) %>
```

Comment:
I was going to gun for that usage

`govuk_back_link_to(manage_ui_course_page_url())`
As those was available in the helper
`params[:provider_code]`
` @course.course_code`

Reply:
We will look at this as a separate change.